### PR TITLE
create navigate

### PR DESCRIPTION
**Pull request by Can-Tina**

### CODE REVIEW

**Review by hannah68 — APPROVED** [review]

**Review by alexamaingard — APPROVED** [review]

File: src/client/pages/signUp/SignUp.jsx [hunk 1]
```diff
@@ -23,8 +27,8 @@ const SignUp = () => {
             body: JSON.stringify(userDetails)
         })
         const data = await res.json()
```
Comment (glowkeeper) [inline src/client/pages/signUp/SignUp.jsx:29]: Is it possible that `data` does not include a token? If so, could you do something like:

```
if (data.hasOwnProperty('token') {
  return true;
} else {
  return false
}  
```
...maybe?

**Review by glowkeeper — CHANGES_REQUESTED** [review]
I reckon you should check whether you've successfully registered...

**Review by vherus — APPROVED** [review]
Nice, but there are a few console logs that should probably get removed before this gets merged

**Comment by Can-Tina** [comment]
Fixed

**Pull request merged**